quarkchain/cluster/jsonrpc.py

```diff
@@ -1299,9 +1299,7 @@ async def getTotalBalance(
         total_balance, next_starter = result
         return {
             "totalBalance": quantity_encoder(total_balance),
-            "next": data_encoder(
-                next_starter + full_shard_key.to_bytes(4, byteorder="big")
-            ),
+            "next": data_encoder(next_starter),
```

Comment:
Now the next address cannot be reused directly?

Reply:
it's a recipient, the func parameter is also a recipient